src/Microsoft.AspNetCore.Blazor.Build/targets/Blazor.MonoRuntime.targets

```diff
@@ -0,0 +1,305 @@
+
+      <!-- /obj/debug/netstandard2.0/blazor/linker/ -->
+      <BlazorIntermediateLinkerOutputPath>$(BlazorIntermediateOutputPath)linker/</BlazorIntermediateLinkerOutputPath>
+      
+      <!-- /obj/debug/netstandard2.0/blazor/linker.marker.txt -->
+      <BlazorIntermediateLinkerResultFilePath>$(BlazorIntermediateOutputPath)linker.marker.txt</BlazorIntermediateLinkerResultFilePath>
+
+      <!-- /obj/debug/netstandard2.0/blazor/Index.html -->
+      <BlazorIndexHtmlOutputPath>$(BlazorIntermediateOutputPath)$(BlazorIndexHtmlName)</BlazorIndexHtmlOutputPath>
+    
+      <!-- /obj/debug/netstandard2.0/blazor/Descriptor.xml -->
+      <GeneratedBlazorLinkerDescriptor>$(BlazorIntermediateOutputPath)Descriptor.xml</GeneratedBlazorLinkerDescriptor>
+
+      <!-- /obj/debug/netstandard2.0/blazor/Descriptor.xml -->
+      <BlazorResolvedAssembliesOutputPath>$(BlazorIntermediateOutputPath)ResolvedAssemblies.txt</BlazorResolvedAssembliesOutputPath>
+
+    </PropertyGroup>
+    
+    <ItemGroup Label="Different outputs used through the build">
+      <ProducedLinkerDescriptor Include="$(GeneratedBlazorLinkerDescriptor)" />
+      <LinkerMarkerFileOutput Include="$(BlazorIntermediateLinkerResultFilePath)" />
+      <BlazorIndexHtmlFileOutput Include="$(BlazorIndexHtmlOutputPath)" />
+      <BlazorResolvedAssembliesOutput Include="BlazorResolvedAssembliesOutputPath" />
+    
+  </ItemGroup>
+   
+    <MakeDir Directories="$(BlazorIntermediateOutputPath)" />    
+    
+  </Target>
+
+  <Target Name="ResolveBinaryOutput" DependsOnTargets="CollectLinkerOutputs;GetBaseClassLibrary" />
+
+  <Target Name="CollectLinkerDescriptors"
+          Inputs="@(IntermediateAssembly);@(BlazorLinkerDescriptor);@(ProducedLinkerDescriptor)"
+          Outputs="@(ProducedLinkerDescriptor)"
+          Condition="$(BlazorShouldRunMonoLinker) == 'true' and '@(BlazorLinkerDescriptor)' == ''">
+    
+    <ItemGroup>
+      <PrepareAssemblyLine Include="@(IntermediateAssembly->'%(FileName)')" />
+      <GeneratedDescriptorLine Include="&lt;linker&gt;" />
+      <GeneratedDescriptorLine Include="@(PrepareAssemblyLine->'&lt;assembly fullname=&quot;%(Identity)&quot; /&gt;')" />
+      <GeneratedDescriptorLine Include="&lt;/linker&gt;" />
+    </ItemGroup>
+
+    <WriteLinesToFile
+      Condition = "'@(BlazorLinkerDescriptor)' == ''"
+      Lines = "@(GeneratedDescriptorLine)"
+      File ="$(GeneratedBlazorLinkerDescriptor)" Overwrite="true" />
+    
+    <ItemGroup Condition = "'@(BlazorLinkerDescriptor)' == ''">
+      <BlazorLinkerDescriptor Include="$(GeneratedBlazorLinkerDescriptor)" />
+      <FileWrites Include="$(GeneratedBlazorLinkerDescriptor)" />
+    </ItemGroup>
+  
+  </Target>    
+    
+  <Target
+    Name="RunMonoLinker"
+    Condition="$(BlazorShouldRunMonoLinker) == 'true'"
+    DependsOnTargets="CollectLinkerDescriptors"
+    Inputs="@(IntermediateAssembly);
+      @(ReferenceCopyLocalPaths->WithMetadataValue('Extension','.dll'));
```

Comment:
This is another case where you may need to have the filenames of the inputs. If you remove a reference that should trigger a re-link

Reply:
All this is removed